roles/pulp_all_services/meta/main.yml

```diff
@@ -28,3 +28,7 @@ dependencies:
   - pulp_services
   - pulp_health_check
   - pulp_webserver
+collections:
+  - ansible.posix
+  - community.crypto
+  - community.postgresql
```

Comment:
This is incorrect. pulp_all_services is just an empty meta role. So is pulp_services.

The dependencies should be in the actual roles that need them. Probably all pulp_common.

Reply:
`community.crypto` is already at `pulp_webserver`role, but it breaks!
After I added it here, it worked, so I think it is related to dynamically including  `pulp_all_services`.
I'll try to move to `pulp_common` and see how it goes

Reply:
moving to `pulp_common` didn't fix the issue:
```
TASK [pulp.pulp_installer.pulp_webserver : include_tasks] **********************
fatal: [pulp3-source-centos8-fips]: FAILED! => {"reason": "couldn't resolve module/action 'x509_certificate'. This often indicates a misspelling, missing collection, or incorrect module path.\n\nThe error appears to be in '/home/faguiar/.ansible/collections/ansible_collections/pulp/pulp_installer/roles/pulp_webserver/tasks/generate_tls_certificates.yml': line 30, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n    - name: Generate CA certificate\n      ^ here\n"}
```

Reply:
That's horrible.

It looks like it only installs the collection dependencies for the role that you explicitly specify.

Let's overcome this by putting pulp_webserver's dependencies in:
1. pulp_webserver
2. pulp_services (with a comment on why)
3. pulp_all_services  (with a comment on why)

Since users may specify 2 and 3 (meta roles) instead of 1.

Reply:
Similarly, pulp_common's dependencies need to be in every role that depend on it (api, content, resman, worker and webserver)

Reply:
pulp_webserver already had the dependencies declared, pulp_services doesn't depend on roles that need collection, pulp_common doesn't depend on any role.
So I've just added a comment on pulp_all_services role

Reply:
Perfect!